src/components/blogs/sections/BlogsDisplay.jsx

```diff
@@ -1,34 +1,59 @@
 import React from "react";
+import { compose } from "redux";
+import { connect } from "react-redux";
+import { firestoreConnect } from "react-redux-firebase";
 import BlogDisplay from "./BlogDisplay";
 import { Link } from "react-router-dom";
 import "../index.css";
 
-const BLOG_POST_SPECS = [
-  {
-    photo: require("././images/blog0.jpg"),
-  },
-  {
-    photo: require("././images/blog1.jpg"),
-  },
-  {
-    photo: require("././images/blog2.jpg"),
-  },
-  {
-    photo: require("././images/blog3.jpg"),
-  },
-];
+// const BLOG_POST_SPECS = [
```

Comment:
commented code fix everywhere please

Reply:
sorry for that, it was another teammate's work. I'll delete it immediately